patterns/1-initial/good-first-project.md

```diff
@@ -0,0 +1,81 @@
+## Title
+
+Good First Project
+
+## Patlet
+
+This pattern presents a framework (_Project-Process-People_ model) and an _accompanying tool_ as one specific implementation of the framework for identifying the _fitness_ of an InnerSource project. The framework is based on prior work by [Klaas-Jan Stol and Brian Fitzgerald](https://ulir.ul.ie/bitstream/handle/10344/4443/Stol_2014_inner.pdf?sequence=2) and [Silona Bonewald](https://innersourcecommons.org/assets/files/InnerSourceChecklist.pdf), and should help facilitate the identification of the most appropriate candidate InnerSource projects and the assessment of InnerSource-readiness of candidate projects.
+
+## Problem
+
+While any project has the potential to benefit from being developed in InnerSource style, some are more suitable candidates than others. While starting with InnerSource, especially when trying to convince the skeptics about the utility of InnerSource (see the [Start as Experiment](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/start-as-experiment.md) pattern) it is important to be able to select the more appropriate projects to have a best chance of success.
+
+Moreover, while previous works focused on the criteria for readiness for InnerSource adoption at an organizational level, there seems to be no concise and actionable guideline for InnerSource-readiness for individual projects.
+
+## Story (optional)
+
+## Context
+
+An organization is starting or is in the early phase of InnerSource adoption, with a lot of skepticism about whether or not it'll work, and selecting an inappropriate pilot project might lead to failure of the entire initiative (see the [Start as Experiment](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/start-as-experiment.md) pattern). Otherwise, a team might be planning to start an InnerSource project but they do not know how prepared they are and if the project they are putting forward is appropriate or not.
```

Comment:
```suggestion
- An organization is starting or is in the early phase of InnerSource adoption, with a lot of skepticism about whether or not it'll work.
- Selecting an inappropriate pilot project might lead to failure of the entire initiative (see the [Start as Experiment](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/start-as-experiment.md) pattern). 
- Otherwise, a team might be planning to start an InnerSource project but they do not know how prepared they are and if the project they are putting forward is appropriate or not.
```

I would break these up in to bullets, just like in the "Forces" section. That way it is easier to check if this is the Context that they also have at their org.

About the last point (starting from "Otherwise, ..."), I am not clear if that is a point in it's own right, or if it belongs to one of the previous points?

Reply:
I agree with the suggestion.

About the last point, what I'm trying to refer to is that teams who are not in the early phase can also assess their project's fitness and identify areas that need attention.